### DISCUSSION TITLE
Is there a docker image for Android SDK version 32?

### DISCUSSION THREAD

**Discussion by DS19971** [mindrunner/docker-android-sdk#86] [General]
We have upgraded our code and now it expects Android SDK 32 to perform the build and it appears their is no docker image for Android SDK 32. If there is a docker image for Android-32 where can I get that?

**Comment by mindrunner** [comment]
Should be supported. Any more information about the error you get?

**Comment by DS19971** [comment]
This is the error we are getting. "2022-12-08T13:05:28.2776940Z Error response from daemon: manifest for docker-hub.artifactory-edge/androidsdk/android-32:latest not found: manifest unknown: The named manifest is not known to the registry."  We are trying to pull it from this link on  Docker-hub https://hub.docker.com/u/androidsdk. If the docker image for Android-32 is available then where can we pull it from?

**Comment by mindrunner** [comment]
It looks like you are in a wrong issue tracker here. My images are available as `docker-android-sdk` at docker hub. See readme for supported tags!

**Comment by mindrunner** [comment]
https://hub.docker.com/repository/docker/runmymind/docker-android-sdk
